### PR TITLE
support cypher parameter

### PR DESCRIPTION
**Pull request by czpmango**

### CODE REVIEW

File: example/GraphClientSimpleExample.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
 #!/usr/bin/env python
 # --coding:utf-8--
 
-# Copyright (c) 2020 vesoft inc. All rights reserved.
+# Copyright (c) 2021 vesoft inc. All rights reserved.
```
Comment (Aiee) [inline example/GraphClientSimpleExample.py:4]: should keep it.

Comment (czpmango) [reply example/GraphClientSimpleExample.py:4]: ACK.

File: tests/test_session.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def test_2_reconnect(self):
                 if i == 3:
                     os.system('docker stop tests_graphd0_1')
                     os.system('docker stop tests_graphd1_1')
-                    time.sleep(3)
+                    time.sleep(6)
```
Comment (Aiee) [inline tests/test_session.py:56]: You could keep the sleep time

File: example/GraphClientSimpleExample.py [hunk 3]
```diff
@@ -28,38 +26,72 @@
...
+        )
+        assert resp.is_succeeded(), resp.error_msg()
+        print_resp(resp)
+        resp = client.execute_parameter(
+            'MATCH (v:person)--() WHERE v.age>abs($p1)+3 RETURN v,v.age AS vage ORDER BY vage, $p3 LIMIT $p1+1',
+            params,
+        )
+        assert resp.is_succeeded(), resp.error_msg()
+        print_resp(resp)
+
```
Comment (Aiee) [inline example/GraphClientSimpleExample.py:94]: This part could be put to the test. Let's keep the example concise.

Comment (czpmango) [reply example/GraphClientSimpleExample.py]: ACK.

File: tests/test_session.py (hunk 2 above)
Comment (czpmango) [reply tests/test_session.py]: ACK.

File: nebula2/gclient/net/Connection.py [hunk 4]
```diff
@@ -198,6 +215,16 @@ def ping(self):
         except Exception:
             return False
 
+    def ping_parameter(self):
+        """check the connection if ok
+        :return: True or False
+        """
+        try:
+            resp = self._connection.execute(0, 'YIELD 1;', None)
```
Comment (jievince) [inline nebula2/gclient/net/Connection.py:223]: execute_parameter

**Review by jievince — APPROVED** [review]

File: nebula2/gclient/net/Connection.py (hunk 4 above)
Comment (czpmango) [reply nebula2/gclient/net/Connection.py]: Already deleted.

**Review by Aiee — APPROVED** [review]

**Pull request merged**